src/main/java/org/teamresistance/frc/Robot.java

```diff
@@ -34,6 +36,8 @@
 
   @Override
   public void robotInit() {
+    opFlow.init();
```

Comment:
Make sure to update this to `IO.myOpticalFlowName.init()` if you're gonna be referencing OFS from `IO`.